docs/src/content/docs/fr/environmental-impact.md

```diff
@@ -92,16 +92,18 @@ Ces tests avec le [Website Carbon Calculator][wcc] comparent des pages similaire
 | Framework                   | CO₂ par page visitée | Notation |
 | --------------------------- | -------------------- | :------: |
 | [Starlight][sl-carbon]      | 0.01g                |    A+    |
-| [Read the Docs][rtd-carbon] | 0.05g                |    A+    |
-| [Sphinx][sx-carbon]         | 0.06g                |    A+    |
+| [Read the Docs][rtd-carbon] | 0.07g                |    A+    |
+| [Sphinx][sx-carbon]         | 0.07g                |    A+    |
 | [VitePress][vp-carbon]      | 0.07g                |    A+    |
-| [Docus][dc-carbon]          | 0.09g                |    A+    |
-| [docsify][dy-carbon]        | 0.10g                |    A     |
-| [Nextra][nx-carbon]         | 0.11g                |    A     |
-| [MkDocs][mk-carbon]         | 0.19g                |    B     |
-| [Docusaurus][ds-carbon]     | 0.21g                |    B     |
-| [GitBook][gb-carbon]        | 0.43g                |    C     |
-| [Mintlify][mt-carbon]       | 1.22g                |    F     |
+| [Docus][dc-carbon]          | 0.10g                |    A     |
+| [docsify][dy-carbon]        | 0.11g                |    A     |
+| [mdBook][md-carbon]         | 0.13g                |    A     |
+| [MkDocs][mk-carbon]         | 0.15g                |    A     |
+| [Fumadocs][fs-carbon]       | 0.16g                |    A     |
+| [Nextra][nx-carbon]         | 0.16g                |    A     |
+| [Docusaurus][ds-carbon]     | 0.25g                |    B     |
+| [Mintlify][mt-carbon]       | 0.99g                |    F     |
+| [GitBook][gb-carbon]        | 1.19g                |    F     |
 
 <small>Données collectées le 22 juillet 2024. Cliquez sur un lien pour voir les chiffres actualisés.</small>
```

Comment:
I think the date also needs updating no?

Reply:
Totally missed it, thanks 🙈